tests/func/parsing/test_interpolated_entry.py

```diff
@@ -259,3 +261,13 @@ def test_vars_load_partial(tmp_dir, dvc, local, vars_):
         d["vars"] = vars_
     resolver = DataResolver(dvc, tmp_dir.fs_path, d)
     resolver.resolve()
+
+
+def test_cmd_dict(tmp_dir, dvc):
+    (tmp_dir / DEFAULT_PARAMS_FILE).dump(CONTEXT_DATA)
```

Comment:
What about non-default params files? Doesn't seem to work with those right now.

Reply:
> What about non-default params files? Doesn't seem to work with those right now.

Not sure I follow, the changes in this P.R. should work the same way as the current templating behavior

Reply:
My misunderstanding